lonelyplanet/rizzo
# Adding in the Criteo JavaScript sent to us by their integration team

## Verification steps

1. Point Waldorf to local Rizzo
2. Bring up any page
3. Verify that a request is sent to the Criteo platform (see picture below)

![screen shot 2015-06-03 at 10 43 02 am](https://cloud.githubusercontent.com/assets/7049277/7964469/76a4cd94-09dd-11e5-9044-4749e5d6c2a3.png)
